src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/HostDocumentComparer.cs

```diff
@@ -1,8 +1,6 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
-#nullable disable
```

Comment:
❗ Need to convert `IEqualityComparer<HostDocument>` to `IEqualityComparer<HostDocument>?`